src/modules/bets.js

```diff
@@ -19,10 +19,46 @@ const parse = str => {
 
 export default function bets(opts) {
 
-  function bet(bot, options) {
+  const { db } = opts
+
+  function bet(bot, options, status = 'open', _entries = {}) {
+    db.schema.createTableIfNotExists('bet_options', (table) => {
+      table.string('name').primary()
+      table.string('represents')
+    })
+      .tap(() => debug('created table bet_options'))
+      .then(() => db('bet_options').del())
+      .then(() =>
+        db('bet_options').insert(Object.keys(options).map(name => {
+          return { name: name, represents: options[name] }
+        }))
+      )
+      .catch(e => { throw e })
+    let __entries
+    db.schema.createTableIfNotExists('bet_entries', (table) => {
+      table.string('user').primary()
+      table.string('option')
+      table.integer('florins')
+    })
+      .tap(() => debug('created table bet_entries'))
+      .then(() => {
+        __entries = Object.keys(_entries).map(lname => _entries[lname])
+        db('bet_entries').del()
```

Comment:
promises need to be `return`-ed from arrow functions with `{}` i.e. `() => { return xx }` (the return is implicit in `() => xx`)